packages/generator/ni_measurement_plugin_sdk_generator/client/templates/measurement_plugin_client.py.mako

```diff
@@ -187,3 +228,11 @@ class ${class_name}:
 
         for response in self._get_stub().Measure(request):
             yield self._deserialize_response(response)
+
+    def register_pin_map(self, pin_map_path: str) -> None:
```

Comment:
We only support registering pin map to the pin map service and not unregistering. Users won't be able to unregister an already registered pin map using this API because we don't have specific grpc calls for unregistering pin map in [pin_map_service.proto](https://github.com/ni/ni-apis/blob/45ef02ecc751cf3aecf7e6130d9ac8efc54f5e0a/ni/measurementlink/pinmap/v1/pin_map_service.proto).

Reply:
ok